### PR TITLE
conf: --with-http_realip_module added

### PR DESCRIPTION
**Pull request by tu1ly**
to be able to use real_ip_header X-Forwarded-For; directive in config to obtain real client ip behind the router

### CODE REVIEW

File: scripts/build_nginx.sh [hunk 1]
```diff
@@ -38,6 +38,7 @@ echo "Downloading $headers_more_nginx_module_url"
 (
 	cd nginx-${NGINX_VERSION}
 	./configure \
+	    --with-http_realip_module
```
Comment (JackuB) [inline scripts/build_nginx.sh]: Broken spacing?

Comment (JackuB) [inline scripts/build_nginx.sh]: Also maybe a missing comment on what this module should achieve

Comment (tu1ly) [inline scripts/build_nginx.sh]: fixed

File: scripts/build_nginx.sh [hunk 2]
```diff
@@ -38,6 +38,7 @@ echo "Downloading $headers_more_nginx_module_url"
 (
 	cd nginx-${NGINX_VERSION}
 	./configure \
+	    --with-http_realip_module \
```
Comment (JackuB) [inline scripts/build_nginx.sh]: snimek obrazovky 2016-12-06 v 16 38 02
Still looks like a wrong spacing and without comment

Comment (tu1ly) [inline scripts/build_nginx.sh]: grrr, looks ok in ide :/

Comment (tu1ly) [inline scripts/build_nginx.sh]: grr, looks ok in editor :/ - fixed

**Pull request merged**